woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -112,6 +112,19 @@ def test_accessor_schema_property(sample_df):
     assert sample_df.ww._schema == sample_df.ww.schema
 
 
+def test_accessor_physical_types_property(sample_df):
+    sample_df.ww.init(logical_types={'age': 'Categorical'})
+
+    assert isinstance(sample_df.ww.physical_types, dict)
+    assert set(sample_df.ww.physical_types.keys()) == set(sample_df.columns)
+    for k, v in sample_df.ww.physical_types.items():
+        logical_type = sample_df.ww.columns[k]['logical_type']
+        if ks and isinstance(sample_df, ks.DataFrame) and logical_type.backup_dtype is not None:
+            assert v == logical_type.backup_dtype
+        else:
+            assert v == logical_type.primary_dtype
```

Comment:
Thoughts on using the `_get_valid_dtype` function here?

Reply:
kind of stuck between wanting to clean this up and also wanting to be a bit more explicit here incase we ever change `_get_valid_dtype` or something to do with dtypes.

Reply:
I don't see this as a big deal either way - we can leave it as it is for now.